README.md

```diff
@@ -1,3 +1,10 @@
+<!--
+GENERATED FILE - DO NOT EDIT
```

Comment:
i can remove the header with config if you prefer

Reply:
Yes, I would prefer not to have that header in the README file. I will give it a shot to add the config.